.github/scripts/run-ini-api-tests.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+from fabric.api import *
+
+from common import manager_fsim_dir, manager_hostname, manager_ci_dir
+from ci_variables import ci_workflow_id
+
+import sys
+
+def run_build_build_recipes_ini_api_tests():
```

Comment:
Duplicate "build" here?

So unless i'm mistaken these seem like they would be slightly annoying to run not as apart of the the CI flow. Can you factor out the salient testing bit of the script from the part that actually dispatches it in CI?

Reply:
I think this script can be run in or out of the CI now since I reworked the CI to run things on the manager self-hosted runners.